src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discount3To6Days = 20;
+  const dailyRentalCost = 40;
+  const discount7OrMoreDays = 50;
```

Comment:
- we never use numbers in the name of our variables
- what if we gonna change the policy and the discount 20 will be applied starting with the second day and ending on the fifth? You gonna need to change the variable name in all the usage places.

So let's think of more reusable names for those variables, more generic